CRM/Contact/Form/DedupeFind.php

```diff
@@ -51,6 +51,7 @@ public function buildQuickForm() {
     $groupList = array('' => ts('- All Contacts -')) + CRM_Core_PseudoConstant::nestedGroup();
 
     $this->add('select', 'group_id', ts('Select Group'), $groupList, FALSE, array('class' => 'crm-select2 huge'));
+    $this->add('text', 'limit', ts('No of contacts to find matches for '));
```

Comment:
Good point - it's definitely not the latter though - it is as many matches are found for the first x number of contacts - so 1000 contacts might return 5 results or 5000.... Not sure the best wording for that.